setup.py

```diff
@@ -5,26 +5,20 @@
 # Author: Zhang Yunjun, Nov 2020                           #
 ############################################################
 
-
+import os
+import sys
 # Always prefer setuptools over distutils
 from setuptools import setup, find_packages
 
+# Grab version and description from version.py
+# link: https://stackoverflow.com/questions/53648900
+sys.path.append(os.path.dirname(__file__))
+from mintpy.version import version, description
```

Comment:
Instead of using this trick you could move to a `setup.cfg` based system (https://setuptools.pypa.io/en/latest/userguide/declarative_config.html), which is declarative and cleaner IMHO.

Reply:
[`setuptools_scm`](https://github.com/pypa/setuptools_scm) also handles all the versioning work being done here implicitly. Only caveat is you need to make sure the git tags have also been fetched at package build time

Reply:
Thanks, `setup.cfg` makes all the sense! I will try to convert it in another PR.